### DISCUSSION TITLE
On boarding PR body isn't updated when the onboarding branch modifies package files

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#17413] [Suggest an Idea]
The onboarding PR is great to see a preview of the detected package files, along with what to expect.

It is great that changes to renovate.json triggers a refresh of the PR.

However, when changes are required the base branch to package files in order to adapt to renovatebot restrictions (such as https://docs.renovatebot.com/modules/manager/flux/ ), it would be convenient to be able to make these changes in the onboarding branch.

I have observed that changes to package files are ignored by the renovatebot process which always clones the base branch.

**Comment by MaronHatoum** [comment]
Hi @gberche-orange 
> The onboarding PR is great to see a preview of the detected package files, along with what to expect.
> 
> It is great that changes to renovate.json triggers a refresh of the PR.
> 
Thank you.

Correct me if I'm wrong the expectation is the `onboarding branch` not only creates a `renovate.json` file but also contains the package files changes and not the base branch.

**Reply by gberche-orange** [reply]
thanks @MaronHatoum I think I observed that packages files changes in the onboarding branch were ignored.
I believe this was illustrated in the job log by the git clone clone being only done against the base branch.

My ask is therefore to have the onboarding "preview" being done on a merge of the onboarding branch into the base branch. Does this makes sense ?
